addons/binding/org.openhab.binding.robonect/src/main/java/org/openhab/binding/robonect/model/MowerStatus.java

```diff
@@ -0,0 +1,66 @@
+package org.openhab.binding.robonect.model;
+
+public enum MowerStatus {
+
+    DETECTING_STATUS(0),
+    PARKING(1),
+    MOWING(2),
+    SEARCH_CHARGING_STATION(3),
+    CHARGING(4),
+    SEARCHING(5),
+    UNKNOWN_6(6),
+    ERROR_STATUS(7),
+    LOST_SIGNAL(8),
+    UNKNOWN_9(9),
+    UNKNOWN_10(10),
+    UNKNOWN_11(11),
+    UNKNOWN_12(12),
+    UNKNOWN_13(13),
+    UNKNOWN_14(14),
+    UNKNOWN_15(15),
+    OFF(16),
+    SLEEPING(17),
+    OFFLINE(98),
```

Comment:
as mentioned before, because of missing access to the things status in rules. however, will be replaced by trigger channel.